var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -173,7 +173,7 @@ def _fix_dtrace_shebang(self, env):
         copy(dtrace, dtrace_copy)
         filter_file(
             "^#!/usr/bin/python",
-            "#!/usr/bin/env {0}".format(os.path.basename(self.spec["python"].command)),
+            "#!/usr/bin/env {0}".format(os.path.basename(str(self.spec["python"].command))),
```

Comment:
We're encouraging people to use f-strings when changing code so you could do that here if you want.